backend/app/services/flow_scheduler.py

```
@@ -0,0 +1,389 @@
+    Get all enabled flows that are due for execution.
+
+    Returns flows where:
+    - is_active = True
+    - enabled = True
+    - next_execution_time <= now
+    - trigger_type is cron, interval, or one_time (not event)
+    """
+    return (
+        db.query(FlowResource)
+        .filter(
+            FlowResource.is_active == True,
```

Ruff (0.14.10)

Avoid equality comparisons to `True`; use `FlowResource.is_active:` for truth checks

Replace with `FlowResource.is_active`

(E712)